webrender_traits/src/types.rs

```diff
@@ -322,20 +349,32 @@ pub struct GlyphKey {
     pub size: Au,
     pub index: u32,
     pub color: ColorU,
+    pub x_suboffset: SubpixelOffset,
+    pub y_suboffset: SubpixelOffset,
```

Comment:
ideally, we'd want to have more variation of the X sub-pixel offsets then Y, right?